frappe/utils/backups.py

```diff
@@ -255,10 +257,10 @@ def get_recent_backup(self, older_than, partial=False):
 			}
 		else:
 			file_type_slugs = {
-				"database": "*-{{}}-{}database.enc.sql.gz".format("*" if partial else ""),
-				"public": "*-{}-files.enc.tar",
-				"private": "*-{}-private-files.enc.tar",
-				"config": "*-{}-site_config_backup.json",
+				"database": "*-{{}}-{}database-enc.sql.gz".format("*" if partial else ""),
+				"public": "*-{}-files-enc.tar",
+				"private": "*-{}-private-files-enc.tar",
+				"config": "*-{}-site_config_backup-enc.json",
```

Comment:
Not really encrypted tho 🤔

Reply:
Ignore this stupidify, I'll fix it separately, it's all broken 😮‍💨